### PR TITLE
SRE miscellaneous changes

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* /bcgov/entity###

*Description of changes:*
1. Cloud Run Jobs - support schedulers, multiple jobs;
2. Clean folders and files;

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: .github/actions/backend-deploy-job/files/cloudbuild.yaml [hunk 1]
```diff
@@ -1,188 +1,251 @@
...
 
-          # replace the yaml
-          gcloud run jobs replace ./devops/gcp/k8s/${JOB_NAME}.yaml --project=${JOB_NAMESPACE} --region=${_REGION}
+        if ! gcloud run jobs describe "${JOB_NAME}" --project="${PROJECT_ID}" --region="${_REGION}" &>/dev/null; then
+          echo "Creating your job..."
+          IMAGE="${_REGION}-docker.pkg.dev/${_BUILD_PROJECT}/cloud-run-repo/${_APP_NAME}:${_DEPLOYMENT_ENV}"
+          gcloud run jobs create "${JOB_NAME}" --project="${PROJECT_ID}" --region="${_REGION}" --image="${IMAGE}"
+        fi
 
+        gcloud run jobs replace ./devops/gcp/k8s/${JOB_NAME}.yaml --project="${PROJECT_ID}" --region="${_REGION}"
```
Comment (seeker25) [inline .github/actions/backend-deploy-job/files/cloudbuild.yaml:218]: Possible we could add in adhoc jobs? I had another PR for that, sometimes OPS needs to run the jobs with edited container settings

**Review by seeker25 — APPROVED** [review]

File: .github/actions/backend-deploy/files/cloudbuild.yaml [hunk 2]
```diff
@@ -198,31 +198,42 @@ steps:
     apt-get update
     apt-get install jq -y
 
-    # Remove the project ID from the release name if it exists
+    # This is done to ensure the release name is consistent across projects
     RELEASE_NAME=$(echo "$RELEASE_NAME" | awk -F'/' '{print $NF}')
     echo "Release name: $RELEASE_NAME"
 
+    # Initialize retry count to 0
+    RETRY_COUNT=0
```
Comment (bolyachevets) [inline .github/actions/backend-deploy/files/cloudbuild.yaml:206]: isn't RETRY_COUNT already defined: https://github.com/bcgov/bcregistry-sre/blob/645811063f18c71c306cf99d48b89c716405d611/.github/actions/backend-deploy/files/cloudbuild.yaml#L195

File: .github/actions/backend-deploy-job/files/cloudbuild.yaml (hunk 1 above)
Comment (pwei1018) [reply .github/actions/backend-deploy-job/files/cloudbuild.yaml:222]: adhoc jobs should be use different way to do it. https://github.com/bcgov/bcregistry-gcp-jobs

File: .github/actions/backend-deploy/files/cloudbuild.yaml (hunk 2 above)
Comment (pwei1018) [reply .github/actions/backend-deploy/files/cloudbuild.yaml]: Fixed

**Pull request merged**